src/power_grid_model/validation/utils.py

```diff
@@ -12,12 +12,15 @@
 import numpy as np
 
 from .. import power_grid_meta_data
+from ..data_types import (
```

Comment:
I think you forgot to commit data_types.py, since the file does not exist in this branch

Reply:
It was added in commit [336809f](https://github.com/alliander-opensource/power-grid-model/pull/106/commits/336809facc421c13150413c2a914b5c51ec35f5e)